src/Components/Address/AddressForm.tsx

```diff
@@ -193,9 +193,11 @@ export const AddressForm: React.FC<AddressFormProps> = ({
         )}
       </Column>
       <Column span={12}>
-        {isAddressAutocompleteEnabled ? (
+        {/* Render the autocomplete input optimistically on the server to make sure we send applicable styles to the client */}
```

Comment:
I don't think this comment makes sense. The reason the styles weren't working was because of an SSR mismatch.